Tools/ros2/README.md

```diff
@@ -231,9 +232,15 @@ speedup:=1 \
 slave:=0 \
 instance:=0 \
 uartC:=uart:./dev/ttyROS1 \
-defaults:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/copter.parm,$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/dds.parm \
+defaults:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/copter.parm,$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/dds_serial.parm \
 sim_address:=127.0.0.1 \
 \
 master:=tcp:127.0.0.1:5760 \
 sitl:=127.0.0.1:5501
 ```
+
+UDP version
+
+```
+ros2 launch ardupilot_sitl sitl_dds_udp.launch.py transport:=udp4 refs:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/dds_xrce_profile.xml synthetic_clock:=True wipe:=False model:=quad speedup:=1 slave:=0 instance:=0 defaults:=$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/copter.parm,$(ros2 pkg prefix ardupilot_sitl)/share/ardupilot_sitl/config/default_params/dds_udp.parm sim_address:=127.0.0.1 master:=tcp:127.0.0.1:5760 sitl:=127.0.0.1:5501
```

Comment:
This works as long as I use mavproxy and set DDS_ENABLE to 1 then reboot. 
`  mavproxy.py --master=udp:127.0.0.1:14550 --console`